packages/api/src/resolvers/importers/uploadImportFileResolver.ts

```diff
@@ -33,14 +33,14 @@ export const uploadImportFileResolver = authorized<
   UploadImportFileSuccess,
   UploadImportFileError,
   MutationUploadImportFileArgs
->(async (_, { type, contentType }, { claims: { uid }, log }) => {
+>(async (_, { type, contentType }, { uid }) => {
   if (!VALID_CONTENT_TYPES.includes(contentType)) {
     return {
       errorCodes: [UploadImportFileErrorCode.BadRequest],
     }
   }
 
-  const user = await userRepository.findOneBy({ id: uid })
+  const user = await userRepository.findById(uid)
```

Comment:
do you need the status here too?

Reply:
@jacksonh i have a custom method `findById` in extended `userRepository` which checks `status` too

Reply:
Awesome, yeah i was looking for something like that but kind of hard to tell in the diff view.